cli/bench/space.go

```diff
@@ -0,0 +1,57 @@
+package bench
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+// Create specified benchmark space with formatting and primary index.
+func createSpace(tarantoolConnection *tarantool.Connection) error {
+	// Creationg space.
+	createCommand := "return box.schema.space.create(...).name"
+	_, err := tarantoolConnection.Exec(tarantool.Eval(createCommand, []interface{}{benchSpaceName, map[string]bool{"if_not_exists": true}}))
+	if err != nil {
+		return err
+	}
+
+	// Formatting space.
+	formatCommand := fmt.Sprintf("box.space.%s:format", benchSpaceName)
+	_, err = tarantoolConnection.Exec(tarantool.Call(formatCommand, [][]map[string]string{
+		{
+			{"name": "key", "type": "string"},
+			{"name": "value", "type": "string"},
+		},
+	}))
+	if err != nil {
+		return err
+	}
+
+	// Create primary index.
```

Comment:
In one place you use -ing (for example ``Foramtting space`` above), in another you don't. Choose one.